index.html

```diff
@@ -623,14 +623,14 @@ <h3>Routing requests</h3>
  <li><p>Remove each entry from <var>endpoints</var>
   for which the concatenation of the <a>URL prefix</a>
   and the entry’s <a>URI template</a>
-  does not match <var>URL</var>’s <a>path</a>.
+  is not equal to <var>URL</var>’s <a>path</a>.
```

Comment:
Ah, well in this particular case it borrows from https://tools.ietf.org/html/rfc6570, which _does_ define matching.